docs/trino-connector/supported-catalog.md

```diff
@@ -122,6 +122,31 @@ call gravitino.system.alter_catalog(
 if you need more information about catalog, please refer to:
 [Create a Catalog](../manage-relational-metadata-using-gravitino.md#create-a-catalog).
 
+## Passing Trino connector configuration
+A Gravitino catalog is implemented by the Trino connector, so you can pass the Trino connector configuration to the Gravitino catalog.
+For example, you want to set the `hive.config.resources` configuration for the Hive catalog, you can pass the configuration to the
```

Comment:
One thing we need to reach a consensus on: does only catalog configuration have the prefix and others like schema/table does not contain the prefix?

Reply:
Only the connector configurations